src/assets/scss/partials/home/image-block.scss

```diff
@@ -19,6 +19,10 @@
 
     }
 
+    .img-container:hover .background-image {
+      opacity: 0.7;
```

Comment:
I wouldn't recommend changing opacity like this on an image. It generally isn't a good thing to do. Instead, I'd recommend using the `:after` pseudo-selector like I showed you in Slack to make a colored overlay which has its own opacity.

If you want the overlay to match the background color of the page section, you can add that via PHP similarly to this: https://github.com/GreaterJacksonHabitat/theme2018/blob/9eb908a5a37df73ffc2de7480643ab6e0e6820c2/front-page.php#L33-L51

Since the Image Block PHP is being `include`'d, it will have access to the `$background_color` variable being used in the loop on the home page.